editor/components/SidebarLayout.tsx

```diff
@@ -86,7 +84,7 @@ export default function SidebarLayout({ children }: { children: ReactNode }) {
           onLogin={() =>
             loginWithRedirect({
               appState: {
-                returnTo: BASE_PATH + pathName,
+                returnTo: pathName,
```

Comment:
☝️ The previous string formatting (`BASE_PATH + pathName`) was wrong. We had never tested it because there was no `onRedirectCallback` function